src/main/java/bio/terra/cli/businessobject/resource/GcsBucket.java

```diff
@@ -150,4 +151,14 @@ public String resolve(boolean includePrefix) {
   public String getBucketName() {
     return bucketName;
   }
+
+  /**
+   * Estimated count of all objects in the bucket
+   *
+   * @return number of objects
+   */
+  public long getObjectCount() {
+    GoogleGcsBuckets buckets = new GoogleGcsBuckets(Context.requireUser().getPetSACredentials());
```

Comment:
same as above: could you add a factory method for this, similar to WSM service class `fromContextForPetSa()`?